.travis.yml

```diff
@@ -1,11 +1,10 @@
 language: python
 python:
-  - "2.6"
+  - "pypy" # Run slowest first so as not to hold up the build
```

Comment:
Travis-CI runs test jobs in parallel, so the order should not matter.

Reply:
It makes a difference if there's more than one build going, as I had with these commits.

If PyPy is the last one to be started, you can be waiting longer to get the result of the whole build.

But I can undo it if it's a problem. :)

Reply:
No, it's not a problem. I just disagree on the conclusion, but I agree on the result.

As far as I can tell, jobs are started depending on capacity - in (for a user) unpredictable order, regardless of this sequence.

So in my understanding this order does not matter at all, so there is no reason for me to object to it.